{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--0248b991-29fc-480a-8e96-1e2344d22bfc",
    "created": "2024-08-13T16:15:53.461879Z",
    "modified": "2024-08-13T16:15:53.461884Z",
    "name": "No Title Available",
    "description": "The rw_vm function in usercopy.c in the 4GB split patch for the Linux kernel in Red Hat Enterprise Linux 4 does not perform proper bounds checking, which allows local users to cause a denial of service (crash).",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2005-2100"
        }
    ],
    "references": [
        {
            "name": "17073",
            "tags": [
                "third-party-advisory",
                "x_refsource_SECUNIA"
            ],
            "url": "http://secunia.com/advisories/17073"
        },
        {
            "name": "RHSA-2005:514",
            "tags": [
                "vendor-advisory",
                "x_refsource_REDHAT"
            ],
            "url": "http://www.redhat.com/support/errata/RHSA-2005-514.html"
        },
        {
            "name": "oval:org.mitre.oval:def:11556",
            "tags": [
                "vdb-entry",
                "signature",
                "x_refsource_OVAL"
            ],
            "url": "https://oval.cisecurity.org/repository/search/definition/oval%3Aorg.mitre.oval%3Adef%3A11556"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=165547"
        }
    ]
}